components/sections/text-and-image.js

```diff
@@ -0,0 +1,24 @@
+// import PropTypes from "prop-types";
+import { Grid } from "@mui/material";
+import style from "../layout/section/section.module.css";
+import Markdown from "react-markdown";
+import Image from "next/image";
+
+export const TextAndImage = ({ data }) => {
+  return (
+    <Grid container spacing={2} columns={8} marginBottom='3rem' marginTop='3rem'>
```

Comment:
this looks great! my only minor suggestions would be increase the spacing in the grid to 6 (`spacing={2} `) to have the image be a bit smaller and let the text take up more of the space (line 10: `sm={2}`, and line 20: `sm={6}`.